### ISSUE TITLE
Can't start DCS SRS Server 1.9.9.0

### ISSUE THREAD

**Issue by Carlton2001** [ciribob/DCS-SimpleRadioStandalone#565]
Updated with SRS-AutoUpdater.exe.
Servers don't start.

```
2021-11-13 21:06:27.3118 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.SRSClientSession | Unable to process JSON: 
 CONNECT 85.206.160.115:80 HTTP/1.1
 Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: C. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at Ciribob.DCS.SimpleRadio.Standalone.Server.Network.SRSClientSession.GetNetworkMessage()
2021-11-13 21:06:27.3118 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.SRSClientSession | Unable to process JSON: 
 Host: 85.206.160.115:80
 Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: H. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at Ciribob.DCS.SimpleRadio.Standalone.Server.Network.SRSClientSession.GetNetworkMessage()
(...)
2021-11-13 21:52:03.4120 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Error receving audio UDP for client A blocking operation was interrupted by a call to WSACancelBlockingCall System.Net.Sockets.SocketException (0x80004005): A blocking operation was interrupted by a call to WSACancelBlockingCall
   at System.Net.Sockets.Socket.ReceiveFrom(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, EndPoint& remoteEP)
   at System.Net.Sockets.UdpClient.Receive(IPEndPoint& remoteEP)
   at Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter.Listen()
2021-11-13 21:52:03.4120 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Error processing Sending Queue UDP Packet: The operation was canceled. 
2021-11-13 21:52:03.4120 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Failed to Process UDP Packet: The operation was canceled. 
2021-11-13 21:52:04.4687 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Error receving audio UDP for client A blocking operation was interrupted by a call to WSACancelBlockingCall System.Net.Sockets.SocketException (0x80004005): A blocking operation was interrupted by a call to WSACancelBlockingCall
   at System.Net.Sockets.Socket.ReceiveFrom(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, EndPoint& remoteEP)
   at System.Net.Sockets.UdpClient.Receive(IPEndPoint& remoteEP)
   at Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter.Listen()
2021-11-13 21:52:04.4697 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Failed to Process UDP Packet: The operation was canceled. 
2021-11-13 21:52:04.4697 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Error processing Sending Queue UDP Packet: The operation was canceled. 
2021-11-13 21:52:05.1682 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Error receving audio UDP for client A blocking operation was interrupted by a call to WSACancelBlockingCall System.Net.Sockets.SocketException (0x80004005): A blocking operation was interrupted by a call to WSACancelBlockingCall
   at System.Net.Sockets.Socket.ReceiveFrom(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, EndPoint& remoteEP)
   at System.Net.Sockets.UdpClient.Receive(IPEndPoint& remoteEP)
   at Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter.Listen()
2021-11-13 21:52:05.1682 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Failed to Process UDP Packet: The operation was canceled. 
2021-11-13 21:52:05.1682 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.UDPVoiceRouter | Error processing Sending Queue UDP Packet: The operation was canceled. 
2021-11-13 22:17:09.7644 | Ciribob.DCS.SimpleRadio.Standalone.Server.Network.ServerState | Unable to read banned.txt System.IO.FileNotFoundException: Could not find file 'C:\Program Files\DCS-SimpleRadio-Standalone\banned.txt'.
File name: 'C:\Program Files\DCS-SimpleRadio-Standalone\banned.txt'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamReader..ctor(String path, Encoding encoding)
   at System.IO.File.InternalReadAllLines(String path, Encoding encoding)
   at Ciribob.DCS.SimpleRadio.Standalone.Server.Network.ServerState.PopulateBanList()
```

Rolled back to 1.9.8.0 and it works

**Comment by dotalchemy** [comment]
Has been reported and resolved - redownload the 1.9.9.0 release :)

**Comment by Carlton2001** [comment]
Indeed, thx =)
